src/_P103_Atlas_EZO_pH_ORP_EC.ino

```diff
@@ -0,0 +1,641 @@
+
+      I2Cchoice = getFormItemInt(F("plugin_103_i2c"));
+      
+      PCONFIG(1) = I2Cchoice;
+
+      PCONFIG_FLOAT(0) = getFormItemFloat(F("plugin_103_sensorVersion"));
+
+      char sensordata[32];
+
+      if (isFormItemChecked(F("Plugin_103_status_led"))) {
+        _P103_send_I2C_command(I2Cchoice, F("L,1"), sensordata);
+      } else {
+        _P103_send_I2C_command(I2Cchoice, F("L,0"), sensordata);
+      }
+      PCONFIG(2) = isFormItemChecked(F("Plugin_103_status_led"));
+
+      String cmd("Cal,");
+      bool   triggerCalibrate = false;
+
+      switch (board_type)
+      {
+          case PH: {
+            PCONFIG_FLOAT(1) = getFormItemFloat(F("Plugin_103_ref_cal_M"));
+            PCONFIG_FLOAT(2) = getFormItemFloat(F("Plugin_103_ref_cal_L"));
+            PCONFIG_FLOAT(3) = getFormItemFloat(F("Plugin_103_ref_cal_H"));
+
+            if (isFormItemChecked(F("Plugin_103_enable_cal_M"))) {
+                cmd             += F("mid,");
+                cmd             += PCONFIG_FLOAT(1);
+                triggerCalibrate = true;
+            } else if (isFormItemChecked(F("Plugin_103_enable_cal_L"))) {
+                cmd             += F("low,");
+                cmd             += PCONFIG_FLOAT(2);
+                triggerCalibrate = true;
+            } else if (isFormItemChecked(F("Plugin_103_enable_cal_H"))) {
+                cmd             += F("high,");
+                cmd             += PCONFIG_FLOAT(3);
+                triggerCalibrate = true;
+            }
+            break;
+          }
+
+          case ORP:
+          {
+            PCONFIG_FLOAT(1) = getFormItemFloat(F("Plugin_103_ref_cal_O"));
+
+            if (isFormItemChecked(F("Plugin_103_enable_cal_O"))) {
+                cmd             += PCONFIG_FLOAT(1);
+                triggerCalibrate = true;
+            }
+            break;
+          }
+
+          case EC:
+          {
+            PCONFIG(3) = getFormItemInt(F("Plugin_103_ref_cal_single"));
+            PCONFIG(4) = getFormItemInt(F("Plugin_103_ref_cal_L"));
+            PCONFIG(5) = getFormItemInt(F("Plugin_103_ref_cal_H"));
+
+            if (isFormItemChecked(F("Plugin_103_enable_cal_dry"))) {
+                cmd             += F("dry");
+                triggerCalibrate = true;
+            } else if (isFormItemChecked(F("Plugin_103_enable_cal_single"))) {
+                cmd             += PCONFIG(3);
+                triggerCalibrate = true;
+            } else if (isFormItemChecked(F("Plugin_103_enable_cal_L"))) {
+                cmd             += F("low,");
+                cmd             += PCONFIG(4);
+                triggerCalibrate = true;
+            } else if (isFormItemChecked(F("Plugin_103_enable_cal_H"))) {
+                cmd             += F("high,");
+                cmd             += PCONFIG(5);
+                triggerCalibrate = true;
+            }
+            break;
+          }
+      }
+
+      if (triggerCalibrate) {
+        char sensordata[32];
```

Comment:
Again creating an array with the same name and signature within the scope where a similar object already exists.
If it is intended, please rename to `sensordata_tmp` or something to make sure it is intended to be a different array.

Reply:
I will check this